### PR TITLE
identity/google: always prompt for user selection.

### LINKED ISSUE #231: [google] Force login choice instead of using an already logged in account
**Describe the bug**

Given:
- an app protected by pomerium configured to use the google identity provider for a given domain
- a user already logged with a google account that is not one of the above domain

When the user accesses the app:
- the authenticate service redirects to google auth page
- google directly consider the user as logged in with the first incorrect identity
- google shows a 403 error saying `Error: org_internal` 

**Expected behavior**

I would expect (if it's actually possible) to have google propose to log into another identity in that case.

**Environment:**

- Pomerium version (retrieve with `pomerium --version`): v0.1.0
- Server Operating System/Architecture/Cloud: Google Cloud Identity Provider

### PR DESCRIPTION
**Pull request by desimone**
Closes #231 

**Checklist**:
- [x] related issues referenced
- [x] updated CHANGELOG.md
- [x] ready for review

### CODE REVIEW

File: internal/identity/google.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func (p *GoogleProvider) Revoke(accessToken string) error {
 // prompt=consent to the OAuth redirect URL and will always return a refresh_token.
 // https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccess
 func (p *GoogleProvider) GetSignInURL(state string) string {
-	return p.oauth.AuthCodeURL(state, oauth2.AccessTypeOffline, oauth2.ApprovalForce)
+	return p.oauth.AuthCodeURL(state, oauth2.AccessTypeOffline, oauth2.SetAuthURLParam("prompt", "select_account consent"))
```
Comment (victornoel) [inline internal/identity/google.go]: @desimone according to https://stackoverflow.com/questions/23436617/force-google-to-show-account-chooser-and-consent-screen, it should be `select_account+consent` (notice the `+` sign)

Comment (victornoel) [reply internal/identity/google.go]: @desimone indeed

**Comment by victornoel** [comment]
@desimone do you know how I can find the tag or the sha of the docker image built for this PR? I can see the build in dockerhub but can't find the info to try it :)

**Comment by desimone** [comment]
@victornoel  try `docker pull pomerium/pomerium:PR232`

**Comment by desimone** [comment]
Thanks @victornoel  👍

**Pull request merged**